index.html

```diff
@@ -145,6 +145,13 @@ <h1 id="title">Eclipse Decentralized Claims Protocol </br>v1.0-RC2</h1>
         specification defines a set of protocols for asserting participant identities, issuing verifiable credentials,
         and presenting verifiable credentials using a decentralized architecture for verification and trust.
     </p>
+    <p>    
+        The Decentralized Claims Protocol (DCP) enables secure, privacy-preserving exchange of verifiable claims across data spaces built on machine-to-machine (M2M) message exchanges. 
+        It leverages decentralized identifiers and verifiable credentials to ensure trust without a central authority, 
+        supporting interoperability between diverse identity systems. Its modular architecture scales across different network sizes, 
+        incorporating policy-based controls and credential revocation. 
+        DCP helps data spaces operate seamlessly, allowing devices, organizations, and individuals to securely manage and verify claims in a trustless environment.
```

Comment:
```suggestion
        DCP helps dataspaces operate seamlessly, allowing participants to securely manage and verify claims in a trustless environment.
```
I don't wanna get into a situation where we'll have to define what a "device" is.